apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html

```diff
@@ -179,31 +179,28 @@
             <div class="col-sm-8 col-xs-12" [ngClass]="{'col-sm-offset-4' : !(currentTxn?.selectedAccount?.stock)}">
                 <div class="text-right mb-1" *ngIf="currentTxn.selectedAccount && showTaxationDiscountBox">
 
-                    <input type="checkbox" [(ngModel)]="blankLedger.isOtherTaxesApplicable"
-                        *ngIf="!blankLedger.isOtherTaxesApplicable"
-                        (ngModelChange)="toggleOtherTaxesAsidePane(blankLedger.isOtherTaxesApplicable)" />
-
-                    <span *ngIf="!blankLedger.isOtherTaxesApplicable">Other Tax ?</span>
-
-                    <div class="clearfix">
-                        <div *ngIf="blankLedger.isOtherTaxesApplicable" class="mrT1 pull-right"
-                            [ngStyle]="{ 'width': totalTdElementWidth + 'px'}">
-                            <label>Other Tax</label>
-                            <span class="d-flex">
-                                <span class="d-flex align-item-center">
-                                    <a href="javascript: void 0" (click)="toggleOtherTaxesAsidePane(true)">
-                                        <img src="assets/images/edit-pencilicon.svg">
-                                    </a>
+                    <ng-container *ngIf="isTcsTdsApplicable">
+                        <input type="checkbox" [(ngModel)]="blankLedger.isOtherTaxesApplicable" *ngIf="!blankLedger.isOtherTaxesApplicable"
+                            (ngModelChange)="toggleOtherTaxesAsidePane(blankLedger.isOtherTaxesApplicable)" />
+                        <span *ngIf="!blankLedger.isOtherTaxesApplicable">Other Tax ?</span>
+
+                        <div class="clearfix">
+                            <div *ngIf="blankLedger.isOtherTaxesApplicable" class="mrT1 pull-right"
```

Comment:
change class name mrT1

Reply:
may be its formatting changes, i checked into file **a month ago comment is added by prateek mishra on that line**